### ISSUE TITLE
Detect continuous/discrete target vals for color util

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#73]
Right now the ParallelCoordinates and RadViz do not work for regressions since it supports a class based visualization. 

Give these classes the opportunity to do a continuous space by using a sequential colormap and coloring the instance lines according to target value.

**Comment by bbengfort** [comment]
Now that we have the `color_sequence`  function, we can create a binning method to assign continuous valued points to the number of classes specified by the colormap.

**Comment by bbengfort** [comment]
Finding a solution to this has been tricky because discrete classes has been embedded into the logic of both RadViz and ParallelCoordinates. There are two primary features that continuous data requires:

1. A colormap that will bin values into a consistent space. 
2. Instead of a label legend, a colorbar that shows the ranges of the instances.

To handle this for now, I'm going to do the following:

- I'm going to create subclasses of ParallelCoordinates and RadViz called "ContinuousParallelCoordinates" and "ContinuousRadViz" respectively. These will be temporary implementations (hopefully) that allow us to use sequential color maps by default. 

I will then create two utility functions: 

- `is_continuous(y)`
- `is_discrete(y)`

Which will accept a vector and use rules to decide if the data is continuous or discrete. Not exactly how to implement this. I will then use these in the `radviz()` and `parallel_coordinates()` helper functions. 

This should solve the issue for now, but we should revisit it when we develop RadViz and ParallelCoordinates with more features and functionality.

**Comment by bbengfort** [comment]
Ok, this ticket has been extremely difficult to work on -- the RadViz and Parallel Coordinates classes are entirely set up for class based visualization only (thanks to their being pulled from matplotlib. 

This needs to be looked at a bit more closely, and I stashed my changes (which touched nearly every module in yellowbrick). We'll have to push this back to another sprint.

**Comment by bbengfort** [comment]
Some notes here:
http://bbengfort.github.io/snippets/2017/01/17/resolving-matplotlib-colors.html

**Comment by bbengfort** [comment]
Sklearn apparently does have a type of target checker that returns continuous or classification as one of it's outputs:

https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/multiclass.py#L175

**Comment by bbengfort** [comment]
A very lightweight version of this has been implemented in #399

**Comment by bbengfort** [comment]
See #334 for package where this should be placed.
